src/util.rs

```diff
@@ -0,0 +1,44 @@
+use log::LevelFilter;
+use simplelog::{CombinedLogger, ConfigBuilder, TermLogger, WriteLogger, TerminalMode};
+use std::error::Error;
+use std::fs::File;
+use std::path::PathBuf;
+use std::convert::Infallible;
+use tokio::sync::mpsc;
+use std::io::Write; 
+
+pub fn get_default_data_dir() -> PathBuf {
+    let home_dir = dirs::home_dir().expect("Home directory not found");
+
+    // Determine the platform-specific path
+    let platform_path = if cfg!(target_os = "linux") {
+        // Path for Linux (Debian/Ubuntu)
+        home_dir.join(".local/share/civkit-node")
```

Comment:
What do you think of `$HOME/.civkitd` as the canonical path for Linux :) ? Though `.local` is good too.

Reply:
updated. makes sense to me to change.